NYCPlanning/ae-cp-map
# Filter projects by cd

Only show capital projects that fall within a community district Highlight the selected community district with its geojson layer

closes #63

## Depends on
API changes made it NYCPlanning/ae-zoning-api#366 

## How to test:

(After setting up the API version this change depends on...)

navigate to a route with the format: 
`boroughs/{boroughId}/community-districts/{communityDistrictId}/capital-projects`

Example:
`/boroughs/2/community-districts/05/capital-projects`

To see how it looks with the community districts layer turned on, add that query parameter:
`/boroughs/2/community-districts/05/capital-projects?districtType=cd`

To see how it behaves with a naive filter menu implementation, check out this experimental branch which let's you navigate via the filter menu: [exp/cp-intersecting-cd](https://github.com/NYCPlanning/ae-cp-map/tree/exp/cp-intersecting-cd)

## Extra
Also syncs kubb generation with `main`

Update deck packages to 9.0.26, as the layer extension was installed with this version
